lib/services/credentials/credential.service.js

```diff
@@ -143,8 +143,21 @@ s.getCredential = function (id, type, options) {
     });
 };
 
-s.getAllCredentials = function (consumerId) {
-  return credentialDao.getAllCredentials(consumerId);
+s.getCredentials = function (consumerId, options = {}) {
+  return credentialDao
+    .getAllCredentials(consumerId)
+    .then((credentials) => {
+      switch (options.mode) {
+        case 'deactivatedOnly':
+          credentials = credentials.filter((credential) => !credential.isActive);
+          break;
+        case 'both':
```

Comment:
`both` is tricky to extend. Imagine you will need some new "mode" parameter, what "both" will mean in this case ? 

`url?include=archived `
`url?include=active, archived `
and default with no params will be `active`

Reply:
What about CLI? -a [--all] for active, archived and --archived for only archived, what do you think @DrMegavolt

Reply:
Or -i [--include]  active, archived

Reply:
I like -i approach